masquerade.go

```diff
@@ -46,6 +46,12 @@ type Provider struct {
 	// Specific hostname mappings used for this provider.
 	// remaps certain requests to provider specific host names.
 	HostAliases map[string]string
+
+	// Allow unaliased pass-through of these domains if no
+	// alias is given. eg "cloudfront.net" for cloudfront
+	// provider.
+	PassthroughDomains []string
```

Comment:
What's the value to explicitly configuring these rather than automatically passing through all unaliased domains?

Reply:
<img width="658" alt="Screen Shot 2019-06-05 at 5 06 01 PM" src="https://user-images.githubusercontent.com/1120829/58990595-43a10580-87b4-11e9-8774-532ef16b9e24.png">
just preserving this request?

Reply:
Ouch, that's sharp! :hocho:

Reply:
I guess a happy medium might be to accept any domains that are valid on that service? Don't really feel strongly and will leave it to your judgment. It just felt like having the passthroughdomains configuration adds a little bit of complexity. Especially since this will have to change as we change distributions, right?

Reply:
Oh not meaning to be mean :D !! I just remember you asked for that hah.  It works as a wildcard suffix which probably needs more explanation or a better name.  So you'd put "cloudfront.net" for all cloudfront.net distributions.

Reply:
I'll make this more explicitly accept wildcards and rename it.

Reply:
Ahh cool, that's actually pretty perfect. Thanks for explaining!

Reply:
okay it now accepts wildcard suffixes like "*.cloudfront.net" or exact matches like "foo.cloudfront.net"